src/test/unittest/ut.c

```diff
@@ -40,6 +40,7 @@
  */
 
 #include "unittest.h"
+#include "util.h"
 
```

Comment:
A date in the copyright header should be updated in this file (src/test/unittest/ut.c).